wasm/tools/fixtures/runner.py

```diff
@@ -79,9 +82,9 @@ def instantiate_module_from_wasm_file(
 
             sub_module = registered_modules[import_.module]
 
-            for export in sub_module["exports"]:
+            for export in sub_module.exports:
                 if export.name == import_.name:
-                    externval = export.desc
+                    externval = export.value
```

Comment:
This was me incorrectly using the `Export` type instead of the `ExportInstance` type.